analytics/src/analytics/integrations/etldb/main.py

```diff
@@ -17,35 +19,38 @@
 VERBOSE = False
 
 
-def init_db() -> None:
+def initialize_database() -> None:
     """Initialize etl database."""
-    # define the path to the sql file
-    parent_path = Path(__file__).resolve().parent
-    sql_path = f"{parent_path}/create_etl_db.sql"
-
-    # read sql file
-    with open(sql_path) as f:
-        sql = f.read()
-
-    # execute sql
-    try:
-        db = EtlDb()
-        cursor = db.connection()
-        cursor.execute(
-            text(sql),
-        )
-        db.commit(cursor)
-    except (
-        RuntimeError,
-        ProgrammingError,
-        OperationalError,
-        InsufficientPrivilege,
-    ) as e:
-        message = f"FATAL: Failed to initialize db: {e}"
-        raise RuntimeError(message) from e
+    # get connection to database
+    etldb = EtlDb()
+    current_version = etldb.get_schema_version()
+    print(f"current schema version: {current_version}")
+
+    # get all sql file paths and respective version numbers
+    sql_file_path_map = get_sql_file_paths()
+    all_versions = sorted(sql_file_path_map.keys())
```

Comment:
Rather than needing to re-sort here. What about either returning tuples or class objects in sorted order?

Something very basic like:
```py
from dataclasses import dataclass

@dataclass
MigrationFile:
    version: int
    file_path: str

```

Reply:
That's another way to do it! :) 
I like the explicit sorting because it shows very clearly, without having to look elsewhere, that the sql files are being executed in order of version.